front/components/poke/data_sources/columns.tsx

```diff
@@ -94,6 +103,12 @@ export function makeColumnsForDataSources(
       id: "actions",
       cell: ({ row }) => {
         const dataSource = row.original;
+        const dataSourceViews = dataSourcesWithViews.find(
+          (v) => v.dataSource.sId === dataSource.sId
+        )?.dataSourceViews;
+        if (!dataSourceViews) {
```

Comment:
should we also handle when `datasourceViews` is empty ?

Reply:
Quite impossible and this is poke really